docs/content/staking/unlocked-staking-guide.mdx

```diff
@@ -1,10 +1,13 @@
 ---
-title: How to Stake with Unlocked FLOW
-sidebar_title: Stake with Unlocked FLOW
+title: Basic Staking with Unlocked FLOW
+sidebar_title: Basic Staking
 ---
 
 This document outlines the steps a token holder can take to stake and manage 
-a Flow node with normal, unlocked FLOW using the `FlowIDTableStaking` contract.
+a Flow node with normal, unlocked FLOW using only the types defined in the `FlowIDTableStaking` contract.
+It only supports having one node or delegator object per account and is not supported by ledger
+and will likely not be supported by other wallets, so it is recommended to use the staking collection
+instead. This style of staking is for those who want to keep their staking operation as simple as possible.
```

Comment:
```suggestion
instead.
```

Sounded like we're deprecating this staking method - not sure we want to give people a reason to keep using it if that's the case.